code/datums/brain_damage/mild.dm

```diff
@@ -265,3 +265,18 @@
 			speak_dejavu += speech_args[SPEECH_MESSAGE]
 	else
 		speak_dejavu += speech_args[SPEECH_MESSAGE]
+
+/datum/brain_trauma/mild/color_blindness
+	name = "Achromatopsia"
+	desc = "Patient's occipital lobe is unable to interpret information from cone cells, rendering the patient completely colorblind."
```

Comment:
```suggestion
	desc = "Patient's occipital lobe is unable to recognize and interpret color, rendering the patient completely colorblind."
```

if you're going for Cerebral Achromatopsia, memories are affected too.

Reply:
this suggestion slightly sucks so if you have a better idea or just want to say it's something else entirely that's okay but i figured i'd just mention it

Reply:
nah its good